tools/preflight/preflight.go

```diff
@@ -774,6 +778,11 @@ func (o *Run) createSnapshotFromPVC(ctx context.Context, volSnapName, volSnapCla
 	pvcName, uid string, clients ServerClients) (*unstructured.Unstructured, error) {
 	volSnap := createVolumeSnapsotSpec(volSnapName, volSnapClass, o.Namespace, snapshotVer, pvcName, uid)
 	if err := clients.RuntimeClient.Create(ctx, volSnap); err != nil {
+		volSnapYAML, yErr := toYAML(volSnap)
+		if yErr != nil {
+			return nil, yErr
+		}
```

Comment:
func `objToYaml` accepts an `interface{}` type as an argument. So why can't we use the same func for both structured and unstructured types.